src/content/security/index.md

```diff
@@ -0,0 +1,231 @@
+
+<Divider />
+
+## Web Security 101
+
+### Use strong passwords
+
+Over 80% of account hacks are a result of weak or stolen passwords. A long combination of characters, numbers and symbols is best to keep your accounts secure.
+
+A common mistake individuals make is using a combination of two to three common, related dictionary words. Passwords like this are insecure because they are prone to a simple hacking technique known as a [dictionary attack](<[https://en.wikipedia.org/wiki/Dictionary_attack](https://en.wikipedia.org/wiki/Dictionary_attack)>).
+
+```md
+Example of a weak password: CuteFluffyKittens!
+
+Example of a strong password: ymv\*azu.EAC8eyp8umf
+```
+
+#### Good password practices:
+
+- Make passwords as long as allowed by either your password generator or the form you're filling out.
+- Use a mixture of uppercase, lowercase, numbers and symbols
+- Avoid common dictionary words
+
+[More on creating strong passwords](https://terranovasecurity.com/how-to-create-a-strong-password-in-7-easy-steps/)
+
+### Use unique passwords for everything
+
+A strong password doesn't provide as much protection if the password is revealed in a data breach. The website [Have I Been Pwned](https://haveibeenpwned.com) allows you to check if your accounts were involved in any data breaches stored in their database. If they have, **you should change pwned passwords immediately**. Using unique passwords for every account lowers the risk of hackers getting access to all of your accounts when one of your passwords is compromised.
+
+### Use a password manager
+
+<InfoBanner emoji=":bulb:">
+  <div>
+    Using a password manager takes care of creating strong, unique passwords and remembering them! We <strong>strongly</strong> recommend using one.
+  </div>
+</InfoBanner>
+
+Remembering strong, unique passwords for every account you have isn't ideal. A password manager offers a secure, encrypted store for all your passwords that you can access through one strong master password. They also suggest strong passwords when signing up for a new service, so you don't have to create your own. Many password managers will also tell you if you have been involved in a data breach, allowing you to change the passwords before any malicious attacks.
+
+![Example of using a password manager](./passwordManager.png)
+
+#### Try a password manager:
+
+- [Bitwarden](https://bitwarden.com/)
+- [KeePass](https://keepass.info/)
+- [LastPass](https://www.lastpass.com/)
+- [1Password](https://1password.com/)
+
+### Use Two-Factor Authentication
+
+To prove you are you there are different unique proofs that can be used for authentication. These are known as factors and the three main factors are:
+
+- something you know (your password)
+- something you are (such as a fingerprint or iris scanner)
+- and something you own (a security key)
```

Comment:
All the other lists are capitalized, this seems to be the only one starting with lowercase letters.